btk/draw_blends.py

```diff
@@ -227,6 +233,7 @@ def render_mini_batch(self, blend_list, cutouts, survey):
             dx, dy = get_center_in_pixels(blend_list[i], wcs)
             blend_list[i].add_column(dx)
             blend_list[i].add_column(dy)
+            # TODO: How to get size for COSMOS?
```

Comment:
That's an issue I encountered too when implementing COSMOS galaxy drawing ; we should check whether the needed parameters are given in the catalog or if we can compute them ourselves. If not, maybe consider to make it optional, so that users may use the MeasureGenerator without computing the size ?